docs/docker_container.md

```diff
@@ -313,20 +313,25 @@ To know more about it, refer to [Version Policy](./model_version_policy.md) docu
 
 
 ### Updating Configuration File
-OpenVINO Model Server, starting from release 2021.1, monitors the changes in its configuration file and applies required modifications in runtime :
+OpenVINO Model Server monitors the changes in its configuration and applies required modifications in runtime in two ways:
 
-- When new model is added to the configuration file config.json, OVMS will load and start serving the configured versions. It will also start monitoring for version changes in the configured model storage. If the new model has invalid configuration or it doesn't include any version, which can be successfully loaded, it will be ignored till next update in the configuration file is detected.
-
-- When a deployed model is deleted from config.json, it will be unloaded completely from OVMS after already started inference operations are completed.
-
-- OVMS can also detect changes in the configuration of deployed models. All model version will be reloaded when there is a change in batch_size, plugin_config, target_device, shape, model_version_policy or nireq parameters. When model path is changed, all versions will be reloaded according to the model_version_policy.
-
-- In case the new config.json is invalid (not compliant with json schema), no changes will be applied to the served models.
-
-**Note**: changes in the config file are checked regularly with an internal defined by the parameter --file_system_poll_wait_seconds.
+- Automatically, with an interval defined by the parameter --file_system_poll_wait_seconds. (introduced in release 2021.1)
+- On demand, by using [Config Reload API](./model_server_rest_api.md#config-reload). (introduced in release 2021.3)  
 
+When configuration reload is triggered in one of above described ways:
 
+- new model or [DAGs](./dag_scheduler.md) added to the configuration file will be loaded and served by OVMS.
+- changes made in the configured model storage (e.g. new model version is added) will be applied. 
+- changes in the configuration of deployed models and [DAGs](./dag_scheduler.md) will be applied. 
+- all model version will be reloaded when there is a change in the model batch_size, plugin_config, target_device, shape, model_version_policy or nireq parameters. When model path is changed, all versions will be reloaded according to the model_version_policy.
```

Comment:
Perhaps we could say that all versions are reloaded when model configuration changed? 
Enumerating all parameters seems to be too much and it's easy to forget/omit some (you didn't mention stateful parameters introduced in this release). We would also need to update it every time we change/add parameters.